src/views/product-downloads-view/components/downloads-section/helpers.ts

```diff
@@ -11,7 +11,7 @@ import {
 export const generateCodeSnippetFromCommands = (
 	commands: PackageManager['commands']
 ): string => {
-	return commands.map((command: string) => `$ ${command}`).join('\n')
+	return commands.join('\n')
```

Comment:
The HDS code block does not expect shell session lines to be prepended with `$ `. If these are added, they are then copied by the Copy button.